test/tools/modlog/converter.js

```diff
@@ -240,6 +240,14 @@ describe('Modlog conversion script', () => {
 				converter.modernizeLog(`[2015-09-19T13:30:44.740Z] (development) ([thetournament] was set to allow scouting by SparksBlade)`),
 				`[2015-09-19T13:30:44.740Z] (development) TOUR SCOUT: by sparksblade: allow`
 			);
+			assert.equal(
```

Comment:
See below - you don't need to test modernizing old modlogs because there are no old logs for this command.